content/en/blog/2023/gc-candidates.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Final list of candidates for the 2023 OpenTelemetry Governance Committee
+linkTitle: 2023 GC Candidates
+date: 2023-10-19
+author: OpenTelemetry Governance Committee
+aliases: [/blog/2023/gc-elections-2023-list-of-candidates]
+cSpell:ignore: Blanco Sirianni Vijay
+---
+
+The OpenTelemetry election committee is pleased to announce the final list of
+candidates running for one of the four available seats. We encourage all voters
+to take a moment and review all candidates, picking the one that best represents
+your interest. You can find their pictures, profile link, and descriptions on
+the
+[candidates page](https://github.com/open-telemetry/community/blob/main/elections/2023/governance-committee-candidates.md),
+but here are their names:
+
+- Austin Parker
+- Daniel Gomez Blanco
+- Johannes Tax
+- Juraci Paixão Kröhling
+- Eric Sirianni
+- Vijay Samuel
+- Severin Neumann
+- Ted Young
+- Steve Flanders
```

Comment:
It should be sorted alphabetically:

- Austin Parker
- Daniel Gomez Blanco
- Eric Sirianni
- Johannes Tax
- Juraci Paixão Kröhling
- Severin Neumann
- Steve Flanders
- Ted Young
- Vijay Samuel

I believe @atoulme is working on a PR to fix the candidates page.

Reply:
By first or last name?

Reply:
Just fix it here if you want, I didn’t start

Reply:
@atoulme its a separate repo so I can't do it in 1 PR

Reply:
@jpkrohling sorted internationally :)